tests/integration/conftest.py

```diff
@@ -201,6 +212,32 @@ def clean_up(self):
         self.remove_keys_and_certificates()
 
 
+@pytest.fixture()
+def fixture_subman(request):
```

Comment:
Maybe the fixture name could be improved, what was wrong with the old name?

Reply:
We did not have this fixture before, so the name is new :D. However, I understand that this doesn't look the best, but I wanted to keep the prefix `fixture` in the name as it correspond to what it actually is and is consistent with the fixture_satellite [here](https://github.com/oamg/convert2rhel/blob/main/tests/integration/conftest.py#L1071). If you have a better name for this, feel free to suggest it